builtin/settingtypes.txt

```diff
@@ -631,7 +631,7 @@ mute_sound (Mute sound) bool false
 
 [*User Interfaces]
 
-#    Set the language. Leave empty to use the system language.
+#    Set the language. Set to (Use system language) to use system language.
```

Comment:
> Since the text will be copied into `minetest.conf.example` when the strings are updated, is the phrase, "Set to (Use system language) ...," which is used in the setting dialog, fine for `minetest.conf`?

The wording proposed by this PR indeed doesn't make sense for `minetest.conf.example`.

What about this?

```suggestion
#    Set the language. By default, the system language is used.
```